lib/Mojo/IOLoop/TLS.pm

```diff
@@ -9,6 +9,9 @@ use Scalar::Util 'weaken';
 use constant TLS => $ENV{MOJO_NO_TLS}
   ? 0
   : eval 'use IO::Socket::SSL 1.94 (); 1';
+use constant DEFAULT => eval { IO::Socket::SSL->VERSION("1.965") }
```

Comment:
We use single quotes whenever possible.

Reply:
right, i actually do that too, don't know why i used double quotes there